index.js

```diff
@@ -1,6 +1,39 @@
 // import your node modules
 
 const db = require("./data/db.js");
-
+const express = require("express");
 // add your server code starting here
-//working???
+
+const server = express();
+
+server.get("/api/posts", (req, res) => {
+  db.find()
+    .then(post => {
+      res.status(200).json(post);
+    })
+    .catch(err => {
```

Comment:
Why use a parameter here when you're not actually using it? Consider empty params as an alternative:
```javascript
.catch(() => {
  // ...
```